CONTRIBUTING.md

```diff
@@ -76,3 +83,18 @@ The first line is the subject and should be no longer than 70 characters, the
 second line is always blank, and other lines should be wrapped at 80 characters.
 This allows the message to be easier to read on GitHub as well as in various
 git tools.
+
+### Format of the Pull Request
+
+We use the pull request title and the first sentence of the pull request description
+to generate the changelog of the next release.
+
+The convention follows the same rules as for commit messages. The PR title reflects the
+what and the first sentence of the PR description reflects the why.
+In most cases one can reuse the commit title as the PR title
+and the commit messages as the PR description for the PR.
+
+If your PR includes more commits spanning mulitple subsystem one should change the PR title
+and the first sentence of the PR description to reflect a summary of all changes involved.
+
+Do not add entries in the changelog yourself. They will be overwritten when creating a new release.
```

Comment:
[release.md](https://github.com/coreos/rkt/blob/master/Documentation/devel/release.md) should be updated (search for "changelog"). For example:

> The CHANGELOG should be updated alongside the code as pull requests are merged into master, so that the releaser does not need to start from scratch.

is not valid anymore.

[release.md](https://github.com/coreos/rkt/blob/master/Documentation/devel/release.md) should request the releaser to run the script.